docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -4,50 +4,251 @@
+
+   `LBWSG PAF notebook <https://github.com/ihmeuw/vivarium_data_analysis/blob/master/pre_processing/lbwsg/LBWSG%20exposure%2C%20rrs%2C%20pafs.ipynb>`__.
+.. todo::
+   - add more description of the all-causes PAF and mode-detailed-cause PAF and the logical reasoning for using one over the other. 
+   - here, we can also discuss the other equations that thought of but did not end up using. 
+
+   
+Assumptions and limitations
++++++++++++++++++++++++++++
+ 
+Strengths
+
+   o  This approach is consistent with GBD methodology and avoids artificially decreasing the mortality rate for individual causes that are not affected by improvements in LBWSG (due to reverse causality or other concerns).
+   
+Limitations
+
+   o  The risk appendix of GBD 2017 says that the data available to compute the relative risks for the risk exposure LBWSG are for the outcome of all-cause mortality. GBD then evaluated the relative risk of all-cause mortality across all available sources.  Based on criteria of biologic plausibility, a list of causes were selected. Some causes, most
+   notably congenital birth defects, haemoglobinopathies, malaria, and HIV/AIDS, were excluded based on the criteria that reverse causality could not be excluded. GBD assumed that the relative risks for all-cause mortality rates by LBWSG category applied equally to mortality rates from each of these blue causes only and did not apply to any other GBD causes in order to calculate the population attributable burden due to LBWSG; in other words, the conservatively ignored the potential impact of LBWSG on mortality due to causes that did not meet their causal criteria. We are choosing to apply the RRs only to this list of LBWSG-affected causes. We believe this is consistent with GBD's approach but may not fully reflect what the RRs capture.
+
+Bias
+
+   Notably, we are not sure which direction this may bias the results (We would need to evaluate stratified microdata results.) If the studies from which we obtain our intervention effect sizes includes mortality data due to causes that GBD considers unaffected by LBWSG, then we may be underestimating the impact of the intervention in our model.
+
+    o   One way of bias could be from not including the reverse-causality causes: suppose we have a nutritional supplement that impacts LBWSG. This supplement was tested in an RCT in western Kenya where malaria is prevalent. Suppose there is some causal link in both directions between birthweight and malaria. While malaria causes lbw, there might also be some causal pathway that lbw babies are more susepticble to malaria infection. If we improve birthweight in this population due to the supplement, we also decrease incidence of malaria, and decrease mortality from malaria. However, this effect through malaria will not be captured in our model, so our modelled effect on neonatal mortality might be less than the empirial effect of this supplement on neonatal mortality. 
```

Comment:
This seems to be glossing over some details a bit, which could be confusing:
> While malaria causes lbw, there might also be some causal pathway that lbw babies are more susepticble to malaria infection.

When we say "malaria causes lbw," presumably we mean malaria in the pregnant _mother_ causes a low birth weight _baby_ (unless you're saying that if a fetus somehow contracts malaria in utero, they are more likely to be low birth weight - but based on some preliminary googling, it seems pretty rare for babies to contract malaria in utero). So I think what's missing is an assumption that malaria in the mother could also cause malaria in the baby, either in utero or, more likely, during the birth (which there seems to be some evidence for).  With that extra assumption, it makes sense that then we wouldn't be able to distinguish which was the cause and which was the effect if a low birth weight baby dies of malaria.  But as written, it is unclear whether we are talking about malaria in the mother or malaria in the baby, which is confusing.

Reply:
what I am saying here is that both have the potential to happen- one is malaria in the mother caused a lbw baby (reverse causality). then the other is that a lbw baby caused it to have malaria (don't know if there is actual evidence for this, but lets say there is...like it makes it more likely the baby will get an infectious bite and develop disease). We do not model malaria as an affected cause but in this case lbw caused malaria and which caused increased mortality. I'll try to clarify a bit.

Reply:
What I'm saying is that "malaria in the mother causes LBW baby" is **not** the reverse of "Being LBW causes the baby to get malaria," because the malaria is happening in two different people.  There are three outcomes to consider here, not just two.  It seems like the only way we can call this "reverse causality" is if we also say that "malaria in the mother causes malaria in the baby."  Or another way to say it might be that malaria in the mother is a confounder for the effect of LBW on malaria in the baby.

Reply:
Yeah. I think we could say something like hemoglobinopathies may cause a baby to be LBWSG AND it is also possible that a neonate is more likely to die from hemoglobinopathies if they are LBWSG.

Reply:
good point, have clarified now!!